# custom messages for publishing and subscribing to visualization data
# of Reachability analysis using Flowstar
#
Header header

string initial
string x
string y
int64 indicator
